### PR TITLE
Switch the program from SIMULATION to Real Operation Mode.

### PR DESCRIPTION
**Pull request by PearTomas**

### CODE REVIEW

**Comment by PearTomas** [comment]
@pavelpetrovic,
Let me confirm. the DeltaT for configuring the temperature offset is already implemented in the code. You can set it through the website in the "PID" section, if I remember correctly.

File: src/controller/V3.0.0/controllerV2/defines.h [hunk 1]
```diff
@@ -14,7 +14,7 @@
 //#define _DHCP set
 
 // simulation
-//#define SIMULATION
+//#define SIMULATION
```
Comment (pavelpetrovic) [inline src/controller/V3.0.0/controllerV2/defines.h:17]: in V3 this is ok, the problem was in V2.

File: src/controller/V3.0.0/controllerV2/defines.h [hunk 2]
```diff
@@ -82,4 +82,4 @@
 // USE ONLY FOR TESTING. BYPASS ALL SAFTY
 #define _DEBUG
```
Comment (pavelpetrovic) [inline src/controller/V3.0.0/controllerV2/defines.h:83]: vaj diz?

**Comment by pavelpetrovic** [comment]
yes, delta T is there, ale Nany su zda sa spat. :|

**Comment by pavelpetrovic** [comment]
mozno to bol iba nejaky celkom zatulany NaN, ktory normalne nechodi. hmm

**Comment by pavelpetrovic** [comment]
tie Nany sa asi zobrazuju iba vo webovom rozhrani ako aktualna teplota ale na server nechodia.
ale to webove rozhranie sa da slusne otvorit iba ked je zariadenie relativne cerstvo resetnute, potom uz to je zakapate...  to kvoli tomu debug, alebo je to ten malloc feature...?

**Comment by pavelpetrovic** [comment]
this was just a chat.

**Pull request closed without merging**